tests/clv/models/test_modified_beta_geo.py

```diff
@@ -0,0 +1,599 @@
+#   Copyright 2024 The PyMC Labs Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+import os
+
+import arviz as az
+import numpy as np
+import pandas as pd
+import pymc as pm
+import pytest
+import xarray as xr
+from lifetimes.fitters.modified_beta_geo_fitter import ModifiedBetaGeoFitter
+
+from pymc_marketing.clv.models.modified_beta_geo import ModBetaGeoModel
+from pymc_marketing.prior import Prior
+from tests.conftest import create_mock_fit, mock_sample
+
+
+class TestModBetaGeoModel:
+    @classmethod
+    def setup_class(cls):
+        # Set random seed
+        cls.rng = np.random.default_rng(34)
+
+        # parameters
+        cls.a_true = 0.891
+        cls.b_true = 1.623
+        cls.alpha_true = 6.282
+        cls.r_true = 0.544
```

Comment:
Test params should match those of `lifetimes`:

https://github.com/CamDavidsonPilon/lifetimes/blob/master/tests/test_estimation.py#L786

Reply:
Done. Tests still pass with no changes, which indicates the functions have no obvious mistakes.

Reply:
Yes - we're mocking a model fit with these test params rather than fitting to the test dataset. We're only fitting to the test data in `test_model_convergence --run-slow`, but I'm glad to see that even the previous params were still within rtol. Might even be able to tighten that tolerance, but it's not required.

Reply:
I reduced rtol by 25%. Unfortunately we cannot reduce further, as the tests start failing